src/axom/inlet/Field.cpp

```diff
@@ -272,21 +273,22 @@ bool Field::get(bool& value)
       "[Inlet] Invalid integer value stored in "
       " boolean value named {0}",
       name());
-    SLIC_WARNING(msg);
+    SLIC_ERROR(msg);
```

Comment:
This seems a bit weird to me...  you are setting an error, which will typically abort, but are also setting a warning flag for when the program continues.

Is this the intended behavior?  Is it handling the case where `slic::setAbortOnError(false)` ?

Reply:
Per Chris's comment about the intended usage of the warning flag, I've removed it from the `get` functions.  That being said, the original intent was for when the program would continue (since these were originally exceptions), but in most cases where I've replaced exceptions with `SLIC_ERROR`, the program will immediately segfault if SLIC does not terminate on error.